kfs-core/src/main/java/edu/arizona/kfs/fp/document/web/struts/YearEndGeneralErrorCorrectionForm.java

```diff
@@ -0,0 +1,14 @@
+package edu.arizona.kfs.fp.document.web.struts;
+
+public class YearEndGeneralErrorCorrectionForm extends GeneralErrorCorrectionForm {
+
+	public YearEndGeneralErrorCorrectionForm() {
+		super();
+	}
+	
+	@Override
+	protected String getDefaultDocumentTypeName() {
+		return "YEGE";
```

Comment:
Fair enough. It always bothers me that foundation deprecates but then doesn't direct to a replacement. One of my biggest pet peeves are magic strings, and I hold a very strong preference for having even just a local class constant to hold these types of things. I know other reviewers have a rule of more than one usage should be moved to a constant, but I haven't been so lenient in the past. From my perspective, it's better to just get in the habit that _all_ magic strings should be extracted out, which if anything, keeps the rule consistent, easy to remember and follow. Have you got a counter argument for leaving it be, aside from slightly less effort? (BTW, logging statements are of course an exception, since logic doesn't hinge on it.)